frappe/core/doctype/report/report.py

```diff
@@ -146,7 +146,7 @@ def execute_query_report(self, filters):
 
 		check_safe_sql_query(self.query)
 
-		result = [list(t) for t in frappe.db.sql(self.query, filters)]
+		result = frappe.db.sql(self.query, filters, as_dict=1)
```

Comment:
ummm, isn't this different?

Reply:
`list[list[Any]] -> [list[dict[str, Any]]`

Reply:
that's the point :laughing: - will update the description in a bit

Reply:
the code automatically handles it down the line btw: https://github.com/frappe/frappe/blob/c973d08fe25bff3bca3c3f1746a5e785ed0788d2/frappe/desk/query_report.py#L127-L134